src/components/books/form.js

```diff
@@ -17,7 +17,7 @@ const Form = () => {
 
   return (
     <form onSubmit={addHandler}>
-       <h1 className="form_title">Add Books</h1>
+       <h1 className="form_title">ADD NEW BOOK</h1>
          <div className="input_ctn">
            <input placeholder="Book Title" className="book_input" name="title"></input>
            <input placeholder="Category" className="category_input" name="category"></input>
```

Comment:
- [ ] Because you are required to style exactly like [Zeplin]() I suggest that you add where to add author name in your form
- [ ] I noticed that you did not work on the categories filter..currently, you can't get categories to choose from while adding a book please fix that

Reply:
The category filter is optional. 

![image](https://user-images.githubusercontent.com/81584449/131923727-ca80904f-b643-40b2-af64-ac485e3a3582.png)

I will put the author's input as you suggested but I think it should be an Optional review.